apps/spruce/src/pages/adminSettings/Tabs.tsx

```diff
@@ -0,0 +1,21 @@
+import styled from "@emotion/styled";
+import useScrollToAnchor from "hooks/useScrollToAnchor";
+
+export const AdminSettingsTab = () => {
+  useScrollToAnchor();
+
+  return (
+    <Container>
+      {/* <Routes>
+        {}
```

Comment:
Try rendering the GeneralTab here.

Reply:
Also add some redirect logs from `/admin-settings` to `/admin-settings/announcements` with the example from [here](https://github.com/evergreen-ci/ui/blob/23cee184a8ebf616e2b7b0533b3fcdb0b46c010c/apps/spruce/src/pages/preferences/PreferencesTabs.tsx#L81-L89)

Reply:
i'm not sure what you mean i'm sorry. where are the redirect logs in the link you sent?